assistant_bot/record_contact.py

```diff
@@ -42,6 +45,26 @@ def find_phone(self, phone: str) -> FieldPhone | None:
             (x for x in self.phones if x.value == phone), None)
 
         return record
+    
+    def add_email(self, email): # email: FieldEmail
+        self.email = email
+        # self.email = FieldEmail(email)
+
+    def edit_email(self, new_email):
+        self.email = new_email
```

Comment:
Please add a "raise NotImplementedError" to all methods that does not have a full implementation.